kythe/cxx/indexer/cxx/IndexerASTHooks.h

```diff
@@ -106,6 +112,7 @@ class IndexerASTVisitor : public RecursiveTypeVisitor<IndexerASTVisitor> {
                     clang::Sema& Sema, std::function<bool()> ShouldStopIndexing,
                     GraphObserver* GO = nullptr, int UsrByteSize = 0,
                     EmitDataflowEdges EDE = EmitDataflowEdges::No,
+                    UseAbsNodes UAN = UseAbsNodes::Abs,
```

Comment:
It is definitely time to move the optional ctor parameters to an Options struct.

Reply:
PTAL